Ginger/Ginger/Actions/ActBrowserElementEditPage.xaml.cs

```diff
@@ -100,8 +104,19 @@ private void SetVisibleControlsForAction()
                     if (mAct.ControlAction == ActBrowserElement.eControlAction.GotoURL)
                     {
                         xOpenURLInPnl.Visibility = System.Windows.Visibility.Visible;
+                        xURLSrcPnl.Visibility = System.Windows.Visibility.Visible;
```

Comment:
for testing that below code didn't broke existing Browser actions:
1- use last Ginger Beta or previous
2- create flow with few Browser actions which has GoToURL operation with few data combinatoins, also add Browser actions with other operations except GoToURL
3- save the solution
4- load the solution with your branch code and confirm that all previously created action loaded correctly and executed as expeted